src/VisualStudio/Core/Test/CodeModel/CSharp/EventCollectorTests.vb

```diff
@@ -786,7 +786,7 @@ class C
 </Code>
 
             Test(code, changedCode,
-                 ArgChange("System.CLSCompliant"))
+                 ArgChange("System.CLSCompliant", "foo"))
```

Comment:
can we use better metasyntactic variables than 'foo' and 'bar'. I don't think we use them in anywhere else in our source :smile: